keyboards/cannonkeys/practice60/config.h

```diff
@@ -18,8 +18,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6464
+#define VENDOR_ID       0xCA04
+#define PRODUCT_ID      0x6060
 #define DEVICE_VER      0x0001
```

Comment:
Sadly github does not allow me to comment on that line, but could you change Line25 to:
`#define MANUFACTURER CannonKeys`